types/bookshelf/index.d.ts

```diff
@@ -373,6 +373,7 @@ declare namespace Bookshelf {
     interface SerializeOptions {
         shallow?: boolean | undefined;
         omitPivot?: boolean | undefined;
+        hidden: array | undefined;
```

Comment:
I think this should be optional:
```suggestion
        hidden?: array | undefined;
```

Reply:
Good, ty!